samples/XAMLCatalog/XAMLCatalog/ProgrammaticViewController.m

```diff
@@ -41,8 +42,11 @@ - (void)viewDidLoad {
     // UIButton
     [self addMenuItemViewController:[[UIButtonViewController alloc] init] andTitle:@"UIButton"];
 
+    // UIButton
+    [self addMenuItemViewController:[[UIButtonWithControlsViewController alloc] init] andTitle:@"UIButton - controls"];
```

Comment:
> - controls [](start = 105, length = 11)

nit: The distinction between this and the above UIButton controller isn't clear.  Can we use a more descriptive name?